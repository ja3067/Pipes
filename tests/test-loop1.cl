out = 0
i = 5

def dynamic():
    if True:
        return 3
    else:
        return 4.0

while i > 0:
    type(out)
    out = out + dynamic()
    i = i - 1
    print(i)
    print(out)
    type(out)

type(out)
print(out)

