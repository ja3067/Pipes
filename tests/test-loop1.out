dyn
dyn
dyn
4
3
3
6
2
9
1
12
0
15
15
